vtex/admin-promotions
# Fix/ch35670/errors feedback

#### What is the purpose of this pull request?
To show alerts and error messages when something weird happens 😱 

#### What problem is this solving?
Lack of error feedbacks on UI.

#### How should this be manually tested?
https://vlauxbeta--pricingqa.myvtex.com/admin/promotions/archived/

All gql mutations in this workspace are intentionally broken. This means that the following actions should show an error message on UI:
* Activate/deactivate promotion
* Archive promotion
* Unarchive promotion
* Save promotion
* Create coupon

#### Types of changes
